luarules/gadgets/cmd_factory_stop_production.lua

```diff
@@ -15,21 +15,7 @@ end
 
 --------------------------------------------------------------------------------
 --------------------------------------------------------------------------------
-
-if not gadgetHandler:IsSyncedCode() then
-	return false --  no unsynced code
-end
-
---------------------------------------------------------------------------------
---------------------------------------------------------------------------------
-
-local spGetRealBuildQueue = Spring.GetRealBuildQueue
-local spGiveOrderToUnit = Spring.GiveOrderToUnit
-local spInsertUnitCmdDesc = Spring.InsertUnitCmdDesc
-
-local CMD_WAIT = CMD.WAIT
-local EMPTY = {}
-local DEQUEUE_OPTS = { "right", "ctrl", "shift" } -- right: dequeue, ctrl+shift: 100
+local magicValue = "StopProduction"
```

Comment:
Is there a more descriptive name for this variable? What does the value represent, the name of the command? Why not use the CMD table and [Spring.GetUnitCmdDescs](https://springrts.com/wiki/Lua_SyncedRead#GetUnitCmdDescs)?

Reply:
The value represents the first argument passed on to `SendToUnsynced` that is usually used as an identifier so the correct gadget receives the information.

Reply:
```suggestion
local identifier = "StopProduction"
```